src/App.jsx

```diff
@@ -14,70 +15,56 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setValue] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const renderActionButton = good => (
+    <button
+      onClick={() => setValue(good === selectedGood ? '' : good)}
+      data-cy={good === selectedGood ? 'RemoveButton' : 'AddButton'}
+      type="button"
+      className={`button ${good === selectedGood ? 'is-info' : ''}`}
```

Comment:
consider to use [classnames library](https://www.npmjs.com/package/classnames) for dynamic build classnames.
```
import cn from 'classnames'
```

```suggestion
      className={cn('button', { 'is-info': isSelected })}
```